web/viewer.js

```diff
@@ -1221,6 +1221,23 @@ var PDFViewerApplication = {
     }
   },
 
+  refreshSidebar: function pdfViewOpenSidebar() {
```

Comment:
Nit: `refreshSidebar` seems like a slightly strange name, since the function only updates the thumbnails. How about `refreshThumbnailViewer` or something similar?